### PR TITLE
Updated readme file + fixed error #1

### PR DESCRIPTION
**Pull request by mast3rz3ro**
Feel free to correct If any typo, thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+- Install Xcode command line tools and [Homebrew](https://brew.sh/)
+- Then with brew install the required packages:
+```
+brew install libxml2 automake
+```
+
+
+## Compiling
+- There is a crude configure.sh to determine which libxml2 to use. To compile, do
+- After you install the required packages, then proceed this way:
```
Comment (kallewoof) [inline README.md:39]: I think you missed removing the above line here.

**Review by kallewoof — COMMENTED** [review]
Looks great, thanks a lot for doing that. One thing noted below.

File: README.md (hunk 1 above)
Comment (mast3rz3ro) [reply README.md:39]: Yes, I was trying to do a minimum changes, thanks!

**Comment by kallewoof** [comment]
Thanks!

**Pull request merged**